scripts/junitReportParser.js

```diff
@@ -0,0 +1,30 @@
+const { XMLParser } = require('fast-xml-parser');
+const fs = require('fs');
+const path = require('path');
+
+const parser = new XMLParser({ ignoreAttributes: false });
+const junitReportFilePath = path.join(__dirname, '../reports/jest-report.xml');
```

Comment:
If you want the parent filepath, try [`path.basename`](https://nodejs.org/api/path.html#path_path_basename_path_ext)

Reply:
how does `path.basename` give me the parent filepath? the docs I read say that i need to give it whole path and it returns the filename. but i want to create the path.